### PR TITLE
Add status codes: 226, 308, 444, 499, 510, 599

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

File: src/Response.php [hunk 1]
```diff
@@ -101,6 +107,7 @@ class Response extends AbstractMessage implements ResponseInterface
         206 => 'Partial Content',
         207 => 'Multi-status',
         208 => 'Already Reported',
+        226 => 'IM used',
```
Comment (mlocati) [inline src/Response.php]: RFC 3229 defines it as `IM Used`, not `IM used`

Comment (samsonasik) [inline src/Response.php]: @mlocati fixed at https://github.com/zendframework/zend-http/pull/110/commits/77b16e23dbde4f1b0379278f21c4be57931406ca , Thanks ;)

File: src/Response.php [hunk 2]
```diff
@@ -138,7 +146,9 @@ class Response extends AbstractMessage implements ResponseInterface
         428 => 'Precondition Required',
         429 => 'Too Many Requests',
         431 => 'Request Header Fields Too Large',
+        444 => 'Connection Closed Without Response',
```
Comment (asgrim) [inline src/Response.php]: Doesn't appear to be "official"? Not sure what the stance is on this.
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#nginx

File: src/Response.php [hunk 3]
```diff
@@ -138,7 +146,9 @@ class Response extends AbstractMessage implements ResponseInterface
...
         451 => 'Unavailable For Legal Reasons',
+        499 => 'Client Closed Request',
```
Comment (asgrim) [inline src/Response.php]: Doesn't appear to be "official"? Not sure what the stance is on this.
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#nginx

File: src/Response.php [hunk 4]
```diff
@@ -149,7 +159,9 @@ class Response extends AbstractMessage implements ResponseInterface
         506 => 'Variant Also Negotiates',
         507 => 'Insufficient Storage',
         508 => 'Loop Detected',
+        510 => 'Not Extended',
         511 => 'Network Authentication Required',
+        599 => 'Network Connect Timeout Error',
```
Comment (asgrim) [inline src/Response.php]: Doesn't appear to be "official"? Not sure what the stance is on this.
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#Unofficial_codes

File: src/Response.php (hunk 2 above)
Comment (samsonasik) [inline src/Response.php]: exists in diactoros/response

File: src/Response.php (hunk 3 above)
Comment (samsonasik) [inline src/Response.php]: exists in diactoros/response

**Comment by samsonasik** [comment]
use iana instead, as proposed in zend\diactoros latest pr at diacoros

**Comment by Xerkus** [comment]
Additional headers are supported by some common web servers. At the same time IANA changes do not happen often and should some be officially assigned we can safely change reason phrase at a later time.

**Comment by samsonasik** [comment]
@Xerkus is there any plan for 2.7.0 release for it ? thank you.

**Comment by samsonasik** [comment]
is there any chance for 2.7.0 release for it ? thank you.

**Comment by samsonasik** [comment]
@Xerkus is there any chance for 2.7.0 release for it ? thank you.

**Comment by Xerkus** [comment]
@samsonasik Sorry, drowned in notifications. I thought i did release it months ago.

**Comment by samsonasik** [comment]
@Xerkus thank you ;)

**Pull request merged**